src/System.Linq.Expressions/tests/InterpreterTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if FEATURE_INTERPRET
+
+using System.Linq.Expressions.Interpreter;
+using System.Reflection;
+using Xunit;
+
+namespace System.Linq.Expressions.Tests
+{
+    public static class InterpreterTests
+    {
+        private static readonly PropertyInfo s_debugView = typeof(LightLambda).GetPropertyAssert("DebugView");
+
+        [Fact]
+        public static void VerifyInstructions_Simple()
+        {
+            Expression<Func<string, bool>> f = s => s != null && s.Substring(1).Length * 2 > 0;
+
+            f.VerifyInstructions(
+                @"object lambda_method(object[])
+                  {
+                    .locals 1
+                    .maxstack 2
+                    .maxcontinuation 0
+                  
+                    IP_0000: InitParameter(0)
+                    IP_0001: LoadLocal(s: 0)
+                    IP_0002: LoadObject(null)
+                    IP_0003: Call(Boolean op_Inequality(System.String, System.String))
```

Comment:
NB: A change was made to `ToString` of the `Call` instruction in order to include the method name. This was inconsistent with e.g. `New` which prints this information. Note that I haven't gone through all `ToString` implementations to assert they produce a meaningful output; as we add tests that assert instruction output, we can revisit `ToString` implementations on an as-need basis.